migrations/V163__Add_Category_Specific_Columns_To_TRB_Admin_Notes.sql

```diff
@@ -0,0 +1,67 @@
+ALTER TABLE trb_admin_notes
+  -- columns for Initial Request Form category
+  ADD COLUMN applies_to_basic_request_details BOOLEAN,
+  ADD COLUMN applies_to_subject_areas BOOLEAN,
+  ADD COLUMN applies_to_attendees BOOLEAN,
+
+  -- columns for Advice Letter category
+  ADD COLUMN applies_to_meeting_summary BOOLEAN,
+  ADD COLUMN applies_to_next_steps BOOLEAN;
+
+-- populate columns for existing admin notes
+UPDATE trb_admin_notes
+SET
+  applies_to_basic_request_details = FALSE,
+  applies_to_subject_areas = FALSE,
+  applies_to_attendees = FALSE
+WHERE category = 'INITIAL_REQUEST_FORM';
+
+UPDATE trb_admin_notes
+SET
+  applies_to_meeting_summary = FALSE,
+  applies_to_next_steps = FALSE
+WHERE category = 'ADVICE_LETTER';
+
+-- add constraints so that only notes of the correct category can set category-specific columns
+
+ALTER TABLE trb_admin_notes
```

Comment:
I found a more concise way of representing these constraints - see https://github.com/CMSgov/easi-app/blob/EASI-3362/sql-constraints-with-manual-test-data/migrations/V167__Add_Category_Constraints.sql. I'm not sure which is more understandable; please share your thoughts.

Reply:
I think the approach you shared in the link best! It feels clearer to me to include both in the one constraint.

Reply:
Updated in https://github.com/CMSgov/easi-app/pull/2254/commits/77caa64a4f1495f6df01aadb5a82b1fbceda10cd.